packages/golden_toolkit/README.md

```diff
@@ -283,6 +283,14 @@ At the moment, it is only possible to load a single .ttf file for a font family.
 
 Additionally, in some instances, it is not possible to replace the "Ahem" font. There are specific places in the Flutter codebase, such as rendering the "debug banner" where no explicit font family is specified. In these instances, the engine will use Ahem in a test context, with no way to override the behavior.
 
+Also note that at the moment for having the required files generated for font processing the pubspec.yaml should have an entry in the images section, make sure it has at least this:
```

Comment:
Can we add a link to both #31  in our repo and the issue in the flutter repo: https://github.com/flutter/flutter/issues/60012